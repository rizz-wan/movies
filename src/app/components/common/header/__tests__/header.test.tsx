import React from "react";
import { Header } from "..";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("App tests", () => {
  describe("Snapshot tests", () => {
    it("Should render app and match snapshot", () => {
      const result = shallow(<Header />);
      expect(result).toBeTruthy();
      expect(result).toMatchSnapshot();
    });
  });
});
