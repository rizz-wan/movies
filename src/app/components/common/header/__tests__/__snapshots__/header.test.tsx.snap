// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests Snapshot tests Should render app and match snapshot 1`] = `ShallowWrapper {}`;
